src/components/Form/index.tsx

```diff
@@ -149,6 +150,55 @@ export const Form = ({
+      ' Denotes a claim indicating that the subject provided assistance, aid, or support to another entity or individual',
+      '  Represents a claim asserting that the subject caused harm, damage, or negative consequences to another entity or individual',
+      ' Indicates a claim suggesting that the subject is involved in fraudulent or deceptive activities.',
+      'Represents a claim asserting ownership or possession of the subject by an individual, organization, or entity',
+      'Represents a claim asserting ownership or possession of the subject by an individual, organization, or entity'
+    ],
+    aspect: [
+      ' Refers to the social impact or influence of the subject.',
+      'Relates to the impact on climate or environmental factors.',
+      ' Relates to the impact on work or employment-related aspects',
+      ' Relates to the impact on financial aspects or economic factors.',
+      'Relates to the impact on education or learning.',
+      'Relates to the technical quality or performance.',
+      ' Relates to the aesthetic or visual quality.',
+      ' Relates to the taste or flavor quality.',
+      'Relates to the journalistic quality or integrity.',
+      'Relates to the academic or educational quality.',
+      'Relates to the fun or entertainment value.',
+      'Relates to the usefulness or practical value.',
+      ' Relates to the literary quality or artistic value.',
+      ' Relates to the relevance or significance.',
+      ' Relates to self-improvement or personal development.',
+      ' Relates to historical significance or relevance.',
+      ' Relates to theological or religious aspects.',
+      ' Relates to adventure or excitement',
+      ' Relates to biographical or personal aspects.',
+      ' Relates to scientific accuracy or validity.',
+      ' Relates to the risk of safety or security concerns.',
+      ' Relates to the risk of reliability or trustworthiness.',
+      ' Indicates a relationship where the subject works for another entity.',
+      'Indicates a relationship where the subject is the same as another entity.'
+    ],
+    howKnown: [
+      'The information is known directly from personal experience or firsthand knowledge.',
+      'The information is known from someone else who has firsthand knowledge or experience',
+      'The information is known from a website as a source',
+      'The information is known from a website that has been verified or authenticated.',
+      'The information is known through a verified login or authentication process.',
+      'The information is known through a signed claim or statement',
+      'The information is known through a blockchain system, providing secure and transparent records',
+      'The information is known from a physical document, such as a paper document or certificate',
+      'The information is known through an integrated system or platform'
+    ]
+  }
```

Comment:
I'd prefer all these strings to be out of the function. If you prefer it in this file, at least take this out of this function. Write this after import but outside the function.